pkg/apis/kubeone/types.go

```diff
@@ -119,6 +119,10 @@ type ClusterNetworkConfig struct {
 	ServiceDomainName string `json:"serviceDomainName"`
 	NodePortRange     string `json:"nodePortRange"`
 	CNI               *CNI   `json:"cni,omitempty"`
+
+	// TODO(xmudrii): Revisit do we want to put this here.
+	// +optional
```

Comment:
Should I keep the `+optional` tag? If yes, should I make this a pointer to string?

Reply:
We should follow here what machine-controller uses (which is []string)